<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigCanais</class>
 <widget class="QDialog" name="ConfigCanais">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>358</width>
    <height>196</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Configuração - Canais</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../imgs/imgs.qrc">
    <normaloff>:/imgs/irc_protocol.png</normaloff>:/imgs/irc_protocol.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Configuração dos canais que serão utilizados para a comunicação.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Planta Real&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Leitura - Tanque 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="real_leitura_t1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Leitura - Tanque 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="real_leitura_t2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Escrita - Tanque 1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="real_escrita_t1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Escrita - Tanque 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="real_escrita_t2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>14</width>
       <height>100</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>38</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt; font-weight:600;&quot;&gt;Simulação&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>38</width>
           <height>17</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>Leitura - Tanque 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="sim_leitura_t1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Leitura - Tanque 2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="sim_leitura_t2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Escrita - Tanque 1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="sim_escrita_t1">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Escrita - Tanque 2</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="sim_escrita_t2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Canal 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 6</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Canal 7</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>25</height>
      </size>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>real_leitura_t1</tabstop>
  <tabstop>real_leitura_t2</tabstop>
  <tabstop>real_escrita_t1</tabstop>
  <tabstop>real_escrita_t2</tabstop>
  <tabstop>sim_leitura_t1</tabstop>
  <tabstop>sim_leitura_t2</tabstop>
  <tabstop>sim_escrita_t1</tabstop>
  <tabstop>sim_escrita_t2</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../imgs/imgs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ConfigCanais</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>224</x>
     <y>176</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ConfigCanais</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>356</x>
     <y>158</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
